lib/kafo/data_types/integer.rb

```diff
@@ -19,7 +19,9 @@ def to_s
       end
 
       def typecast(value)
-        value =~ /\d+/ ? value.to_i : value
+        Integer(value)
+      rescue
```

Comment:
Isn't it better to catch `ArgumentError` explicitly? Catching all exceptions is generally a bad practice.